runtime/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "dscp-node-runtime"
-version = "4.4.3"
+version = "4.4.4"
```

Comment:
@dblane-digicatapult are we keeping the `dscp-node` version is lockstep with the `dscp-runtime` these days? I ask because if it is we need to also bump the `spec_version` and bump the referenced version in the `dscp-node` `Cargo.toml`

Reply:
ok so @nitro-marky we need to update the `spec_version` in the runtime `lib.rs` to be `444` and we need to update the version of the `dscp-runtime` referenced in the `node/Cargo.toml` to be `4.4.4`